coderd/database/migrations/000229_dormancy_notification_template.up.sql

```diff
@@ -0,0 +1,35 @@
+INSERT INTO
+    notification_templates (
+        id,
+        name,
+        title_template,
+        body_template,
+        "group",
+        actions
+    )
+VALUES (
+        '0ea69165-ec14-4314-91f1-69566ac3c5a0',
+        'Workspace Marked as Dormant',
+        E'Workspace "{{.Labels.name}}" marked as dormant',
+        E'Hi {{.UserName}}\n\n' || E'Your workspace **{{.Labels.name}}** has been marked as **dormant**.\n' || E'The specified reason was "**{{.Labels.reason}} (initiated by: {{ .Labels.initiator }}){{end}}**\n\n' || E'Dormancy refers to a workspace being unused for a defined length of time, and after it exceeds {{.Labels.dormancyHours}} hours of dormancy might be deleted.\n' || E'To activate your workspace again, simply use it as normal.',
```

Comment:
The wording is a little weird here. Can we change it and add a link to the dormancy docs:

I don't think we need to include the initiator, just improve the reasons (see end of message).

> Hi `{user-name}`, your workspace `{ws-name}` has been marked [dormant](https://coder.com/docs/templates/schedule#dormancy-threshold-enterprise) because of `{reason}`.

Append if dormancy cleanup is enabled and `dormancy_hours` < 24:
> Dormant workspaces are [automatically deleted](https://coder.com/docs/templates/schedule#dormancy-auto-deletion-enterprise) after `{dormancy_hours}` hours of inactivity.

Append if dormancy cleanup is enabled, and `dormancy_hours` > 24:
> Dormant workspaces are [automatically deleted](https://coder.com/docs/templates/schedule#dormancy-auto-deletion-enterprise) after `{dormancy_hours // 24}` days of inactivity.

> To prevent deletion, use your workspace with the link below.

---

The reasons also fit into differing grammatic context:
- Lifecycle executor: "breached the template's threshold for inactivity"
- API: "requested by user"
- Template: "template updated to new dormancy policy"

If we use these, they'd fit the above messages better.
- Lifcycle: "prolonged inactivity, exceeding the dormancy threshold"
- API: "a user request"
- Template: "an update to the template's dormancy"

Reply:
Going to refactor the templates in the next PR to avoid further postponement.